R/rspt01.R

```diff
@@ -156,7 +156,7 @@ rspt01_post <- function(tlg, prune_0 = TRUE, ...) {
   if (prune_0) {
     tlg <- smart_prune(tlg)
   }
-  std_postprocess(tlg)
+  report_null(tlg)
```

Comment:
table inset is not expose, you would not be able to change it if not in post processing right? if this is on purpose we can lose other inputs in report_null

Reply:
In chevron, we are aiming the creation of standard representation of TLG. In the specification of the  standard, there is this small inset for the table. Should the user want to modify that, they can do it later or edit the postprocessing function, but le me make it easier to pass the argument.